lib/views/pages/profile/user_info/user_account.dart

```diff
@@ -7,10 +7,14 @@ class UserAccount extends StatelessWidget {
 
   const UserAccount({
     super.key,
-    required this.userEmail,
+    required this.userName,
```

Comment:
Here, why not directly pass the `UserData` data class instead of individual attributes ?